native/exe/SageLauncher/launcher.cpp

```diff
@@ -1135,7 +1164,7 @@ int launchJVMSage(LPSTR lpszCmdLine, HWND hWnd, BOOL bClient, BOOL bService)
 		if (RegQueryValueEx(myKey, "JVMMaxHeapSizeMB", 0, &readType, (LPBYTE) &regMemVal, &hsize) == ERROR_SUCCESS)
 		{
 			if (regMemVal > 0)
-				sprintf(memString, "-Xmx%dm", regMemVal);
+				sprintf_s(memString, sizeof(memString), "-Xmx%dm", (int)regMemVal);

```

Comment:
You shouldn't have to cast regMemVal...instead use the proper format type in sprintf instead if it's complaining about that (I think it's %lu that it wants, or maybe %u).